StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Link/LinkPaymentController.swift

```diff
@@ -0,0 +1,221 @@
+//
+//  LinkPaymentController.swift
+//  StripePaymentSheet
+//
+//  Created by Bill Meltsner on 12/9/22.
+//  Copyright © 2022 Stripe, Inc. All rights reserved.
+//
+
+import UIKit
+@_spi(STP) import StripePayments
+
+
+/// `LinkPaymentController` encapsulates the Link payment flow, allowing you to let your customers pay with their Link account.
+/// You probably want to [use the Mobile Payment Element instead](https://stripe.com/docs/payments/accept-a-payment?platform=ios&ui=payment-sheet), which supports Link alongside credit cards, bank transfers, and many other payment methods to offer your customers more choice and convenience in how they pay.
+@available(iOSApplicationExtension, unavailable)
+@available(macCatalystApplicationExtension, unavailable)
+@_spi(LinkOnly) public class LinkPaymentController {
+
+    private var intentSecret: IntentClientSecret
+    private var configuration: PaymentSheet.Configuration
```

Comment:
Do either of these actually need to be vars?

Reply:
Not anymore! Good catch.